wellcomeml/ml/spacy_classifier.py

```diff
@@ -81,23 +83,24 @@ def fit(self, X, Y):
         X_train, X_test, Y_train, Y_test = train_test_split(
             X, Y, random_state=42
         )
+        # Free memory
+        del X
```

Comment:
I think there is an interesting discussion to whether `del` frees memory. Some people found it that hey needed to call the garbage collector. I think we should be good for now, but would be interesting to raise it with some engineer in the future.

Reply:
https://stackoverflow.com/questions/1316767/how-can-i-explicitly-free-memory-in-python - in case we still get into trouble with memory

Reply:
@jdu any thoughts on this?